code/modules/multiz/zshadows.dm

```diff
@@ -0,0 +1,63 @@
+
+
+//An appearance, in movable atom form, designed to show through Zs
+//System only visibly displays 2 z of depth, for visual clarity+performance
+/atom/movable/zshadow
+	var/atom/owner
+
+/atom/movable/zshadow/Initialize()
+	..()
+	LAZYADD(SSz.zshadows, src)
+
+/atom/movable/zshadow/Move() //loc = only bby.
+	return
+
+/atom/movable/zshadow/CanFallThroughZ() //amusing, but no
+	return
+
+/atom/movable/zshadow/Destroy()
+	if(!QDELETED(owner)) //Nothing destroys these if their owner exists, they *ARE* their owner as far as we're concerned
```

Comment:
Almost nothing uses force atm, but we should support it in case we ever do decide we need it